BPA/Ch.06/Mth.stan

```diff
@@ -67,16 +67,22 @@ model {
 }
 
 generated quantities {
+  vector<lower=0,upper=1>[T] p[M];
   int<lower=0,upper=1> z[M];
   int<lower=C> N;
 
   for (i in 1:M) {
+    for (j in 1:T)
```

Comment:
Same comments in these loops as in the previous ones.  And if possible, pulling this out into a shared function as you've done elsewhere might make the structure across models a little easier to see.